docs/tensorstore_demo/__init__.py

```diff
@@ -735,8 +735,8 @@ def __getitem__(self, *args, **kwargs) -> None:
             })
 
         More generally, specifying an ``n``-dimensional `bool` array is equivalent to
-        specifying ``n`` 1-dimensional index arrays, where the ``i``\ th index array specifies
-        the ``i``\ th coordinate of the `True` values:
+        specifying ``n`` 1-dimensional index arrays, where the ``i``\\ th index array specifies
+        the ``i``\\ th coordinate of the `True` values:
```

Comment:
I'm guessing this is done to resolve warnings about invalid escape sequences. I hadn't thought that we'd need to escape the escaping character in a literal string used as a doc string. It makes sense though.

I'll have to remember this as it still seems to [render as expected in the docs](https://sphinx-immaterial--366.org.readthedocs.build/en/366/python_apigen_generated/DimExpression.__getitem__.html#boolean-array-indexing).

Reply:
Python ignores invalid escape sequences, so they still work correctly, but I wanted to avoid the warnings.